src/cosmicqc/scdataframe.py

```diff
@@ -71,8 +71,13 @@ def __init__(
             self.data_source = "pandas.DataFrame"
             self.data = data
 
+        elif isinstance(data, pd.Series):
+            # if data is a pd.DataFrame, remember this within the data_source attr
+            self.data_source = "pandas.Series"
+            self.data = pd.DataFrame(data)
+
         elif isinstance(data, (pathlib.Path, str)):
-            # if the data is a string or a pathlib path, remember the original source
+            # if the data is a string, remember the original source
```

Comment:
I remember this being included in a different PR for clarity, is there a reason to remove the pathlib path part?

Reply:
Thanks for checking - no, sorry, it's a mistake on my part during merge conflict resolutions.